src/edu/ucsb/nceas/metacat/properties/PropertiesWrapper.java

```diff
@@ -281,6 +301,22 @@ protected Path getMainMetadataFilePath() {
         return mainMetadataFilePath;
     }
 
+    /**
```

Comment:
FYI, I refactored this to here form PropertyService, since I call the same code here from within initialize(), so this makes it simpler and follows DRY